update/src/worker.ts

```diff
@@ -0,0 +1,21 @@
+// @@@SNIPSTART typescript-update-worker
+import { NativeConnection, Worker } from '@temporalio/worker';
+
+async function run() {
+  const connection = await NativeConnection.connect({
+    address: 'localhost:7233',
+  });
+  const worker = await Worker.create({
+    connection,
+    namespace: 'default',
+    taskQueue: 'my-task-queue',
```

Comment:
Use a task queue that's unique for this sample. In case users run multiple samples, you want their workers to not collide.